app/partials/actions/add.html

```diff
@@ -1,10 +1,10 @@
 <div class="container">
 	<h3>
-		Add {{rdesc.name.singular}}
+		Add {{rdesc.attributes.name.singular}}
 	</h3>
-	<h5>{{rdesc.description}}</h5>
+	<h5>{{rdesc.attributes.description}}</h5>
 
-	<div class="form-group" ng-repeat="field in rdesc.fields" ng-include="'/partials/fields/default-field.html'">
+	<div class="form-group" ng-repeat="field in rdesc.attributes.fields" ng-include="'/partials/fields/default-field.html'">
```

Comment:
Great! 

I put it in a dasherized format because I've actually been thinking about updating all the keys in the auto-generated documentation (if not those in our models themselves), to use dasherized names per [this recommendation](http://jsonapi.org/recommendations/#naming). But that's a separate conversation and not super important.